CHANGELOG.md

```diff
@@ -3,6 +3,157 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] nfcore/rnafusion - 2022/03/??
+
+Update to DSL2 and newer software/reference versions
+
+### Non-DSL2 related changes
+
+- replace gtf file with chr.gtf: Please confirm if this change is ok for the whole pipeline
+- remove subdirectory with params.genome as only GRCh38 is supported (not necessary but helped me see the output more clearly, can be reversed)
+- modification of output parameters to make sure every important file is published
+- additional build_references flag to trigger the workflow to build references
+- container for starfusion is changed and the absolute path to prep_genome_lib.pl is given
+  --fasta argument is not required when executing the workflow to build references
+- increase cpu + memory for process STARFUSION_DOWNLOAD
+- ci test done on stubs of reference building subprocesses ensembl and arriba
+- **For the moment, the rnafusion workflow (to analyse data and not build the references) is removed as dependencies and links would have to be fixed in the rnafusion workflow for execution of the build references otherwise**
```

Comment:
This sentence could use some clarification.

Reply:
Addressed in 7bad769